Examples/Python/python/acts/examples/track_jet.py

```
@@ -0,0 +1,32 @@
+from typing import Optional, Union, Any, List
```

Ruff (0.8.2)

`typing.Union` imported but unused

Remove unused import

(F401)